README.md

```diff
@@ -0,0 +1,257 @@
+
+For now, we're using the [GitHub Container Registry](https://ghcr.io) (which is
+an [OCI registry](https://helm.sh/docs/topics/registries/)) to host our Helm
+chart. Helm 3 has _experimental_ support for OCI registries. In the event that
+the Helm 3 dependency proves troublesome for users, or in the event that this
+experimental feature goes away, or isn't working like we'd hope, we will revisit
+this choice before going GA.
+
+To fetch the Brigade Docker Hub Gateway chart from the registry:
+
+```console
+  export HELM_EXPERIMENTAL_OCI=1
+  helm chart pull ghcr.io/brigadecore/brigade-dockerhub-gateway:v2.0.0-alpha.1
+  helm chart export ghcr.io/brigadecore/brigade-dockerhub-gateway:v2.0.0-alpha.1 -d ~/charts
+```
+
+As this chart requires custom configuration as described above to function
+properly, we'll need to create a chart values file with said config.
+
+Use the following command to extract the full set of configuration options into
+a file you can modify:
+
+```console
+$ helm inspect values ~/charts/brigade-dockerhub-gateway > my-values.yaml
+```
+
+Edit `my-values.yaml`, making the following changes:
+
+* `brigade.apiAddress`: Address of the Brigade API server, beginning with
+  `https://`
+
+* `brigade.apiToken`: Service account token from step 2
+
+* `tokens`: This field should define tokens that can be used by clients to send
+  events (webhooks) to this gateway. Note that keys are completely ignored by
+  the gateway and only the values (tokens) matter. The keys only serve as
+  recognizable token identifiers for human operators.
+
+Save your changes to `my-values.yaml` and use the following command to install
+the gateway using the above customizations:
+
+```console
+$ helm install brigade-dockerhub-gateway ~/charts/brigade-dockerhub-gateway \
+    --create-namespace \
+    --namespace brigade-dockerhub-gateway \
+    --values my-values.yaml
+```
+
+### 3. (RECOMMENDED) Create a DNS Entry
+
+If you installed the gateway without enabling support for an ingress controller,
+this command should help you find the gateway's public IP address:
+
+```console
+$ kubectl get svc brigade-dockerhub-gateway \
+  --namespace brigade-dockerhub-gateway \
+  --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+If you overrode defaults and enabled support for an ingress controller, you
+probably know what you're doing well enough to track down the correct IP without
+our help. 😉
+
+With this public IP in hand, edit your name servers and add an `A` record
+pointing your domain to the public IP.
+
+### 4. Create Webhooks
+
+In your browser, go to your Docker Hub repository for which you'd like to send
+webhooks to this gateway. From the tabs across the top of the page, select
+__Webhooks__. 
+
+* In the __Webhook name__ field, add a meaningful name for the webhook.
+
+* In the __Webhook URL__ field, use a value of the form `https://<DNS hostname or publicIP>/events?access_token=<url-encoded token>`.
```

Comment:
This URL takes a url-encoded token whereas the `tokens` chart field appears to take non-encoded token values.  If this is correct, should we add guidance on how to url-encode a token from the `tokens` field for building the URL here?  (Or, if not how, maybe just a mention that this needs to be done.)

Reply:
I'll add a note to values.yaml about this.